tinker/templates/faculty-bios/home.html

```diff
@@ -57,6 +68,21 @@
                         </a>
                     </li>
                 {% endif %}
+                {% if show_courseleaf_button %}
+                    {% if bio['courseleaf-user'] == 'Yes' %}
+                        <li>
+                            <a id="{{ bio.id }}" data-method="No" data-faculty-bio-id="{{ bio.id }}"
+                               class="small btn btn-primary btn-alert courseleaf-user-press">Remove from Catalog
+                            </a>
+                        </li>
+                    {% else %}
+                        <li>
+                            <a id="{{ bio.id }}" data-method="Yes" data-faculty-bio-id="{{ bio.id }}"
+                               class="small btn btn-primary btn-alert courseleaf-user-press">Add to Catalog
+                            </a>
+                        </li>
```

Comment:
Since this IF block is nearly identical, try to pull out what you can. At least the "<li>"'s.

Reply:
Pulled out the li's